Cookie policy

nenadV91 opened the pull request:
# Summary

Adds Cookie policy page based on [this file
](https://docs.google.com/document/d/1_G9YNuLxLhosWBAmNmHxgwFjeGlUCH6iDpl_IuPppYo/edit)

# To test
1. scroll down to the footer
2. click on Cookie policy link
3. you should see the cookie policy page

elena-zh reviewed (changes requested):
Hey @nenadV91 , great job!
Issues: 
1. there is a duplicating sentence: 
![double text](https://user-images.githubusercontent.com/70885163/234255334-80f03b43-7f4c-4b89-be18-8365b839b128.png)
2. In this paragraph there should be commas instead of semicolons:
![comma and period](https://user-images.githubusercontent.com/70885163/234255537-0f62cfd4-a3c4-453f-9356-04b663403b90.png)
3. And it would be great to open links in new tabs
4. I don't think that the policy text should be center-aligned in a mobile view. Left aligned text looks better to me. WDYT?

Thanks!
![image](https://user-images.githubusercontent.com/70885163/234256116-51433122-3b87-464f-9880-123122e82834.png)

nenadV91 commented:
@elena-zh 
1. Great catch, fixed
2. I think it is fine like that, we are doing the same on https://swap.cow.fi/#/cookie-policy?chain=mainnet
3. Unfortunately seems that we can't do this here. In cowswap we used other plugin to parse markup and we could use html in it, but here we can't do that.
4. Good suggestion, updated.

elena-zh reviewed (approved):
LGTM then.

shoom3301 commented on src/const/meta.ts:
```diff
@@ -22,6 +22,7 @@ export const CONFIG = {
     cowProtocol: "https://cow.fi",
     beaver: "https://beaverbuild.org/",
     agnostic: "https://agnostic-relay.net/",
+    cookiePolicy: '/cookie-policy'
```
Mixed single/double quotes, I think we should use only one

nenadV91 replied:
Good catch 👍
